src/cloudai/_core/base_runner.py

```diff
@@ -69,7 +69,7 @@ def __init__(
         self.mode = mode
         self.system = system
         self.test_scenario = test_scenario
-        self.output_path = self.setup_output_directory(system.output_path)
+        self.output_path = self.setup_output_directory(test_scenario.name, system.output_path)
```

Comment:
We already have `self.test_scenario` wouldn't it be better to access it from `setup_output_directory`? That would keep the API unchanged.

Reply:
If we want to give users more control over output directory name, let's add a separate function for that and `setup_output_directory` will use that function (or property).

Reply:
Updated.